challenges/victor/backEndChallenge/routes/index.js

```diff
@@ -0,0 +1,39 @@
+var express = require("express");
+var router = express.Router();
+const { check, validationResult } = require("express-validator");
+
+var data;
+
+router.get("/data", (req, res) => {
+  res.json(data);
+});
+
+router.post(
+  "/data",
+  [
+    check("numbers")
+      .isArray({ min: 500, max: 500 })
+      .withMessage("You must submit a list of 500 numbers exactly."),
+    check("numbers")
+      .custom(
+        arr =>
+          !arr.some(index => {
+            !Number.isInteger(index);
```

Comment:
Is the client not able to submit floating point numbers?

Reply:
Currently, the client can only send floats that are whole numbers or don't have any decimal places. To allow decimal places I could change the function to check the typeof index and return true if it's a number and false if it's anything else. (I would have to also create exceptions for infinity and NaN that return false as they read as number types).